scripts/datacontainer/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM alpine:3.8
+FROM mcr.microsoft.com/oss/mirror/docker.io/library/ubuntu:20.04
```

Comment:
where do you get this location? does it allow public access?

Reply:
Yes. https://eng.ms/docs/more/containers-secure-supply-chain here is link.

Reply:
Can you use the other one for alpine - mcr.microsoft.com/cbl-mariner/distroless/base:1.0?

We would like to keep this image as small as possible.

Reply:
Fixed.